src/app/resources/resources-add.component.ts

```diff
@@ -195,6 +195,7 @@ export class ResourcesAddComponent implements OnInit {
           (resourceRes) => this.afterResourceUpdate(message, resourceRes),
           (err) => this.planetMessageService.showAlert('There was an error with this resource')
         );
+        this.resourcesService.sendResourceNotification(newResource);
```

Comment:
After changing the function to `return forkJoin...`, you should add this in another `switchMap` in the pipe for `updateResource`